Sources/BraveWallet/Crypto/NFTImageView.swift

```diff
@@ -0,0 +1,47 @@
+/* Copyright 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import SwiftUI
+import BraveUI
+import SDWebImageSwiftUI
+
+struct NFTImageView<Fallback: View>: View {
+  let urlString: String
+  
+  private var fallback: () -> Fallback
```

Comment:
nit: Fallback name is a bit odd as this is used when loading not just fallback after fail to load. Maybe `placeholder` to match the extension / modifier on `WebImage`

Reply:
fixed in [00c5f80](https://github.com/brave/brave-ios/pull/6534/commits/00c5f80308e952e4dc6914ae61d3bbcddda7d726)